src/vl-search-filter.js

```diff
@@ -13,6 +13,7 @@ import '/node_modules/vl-ui-titles/dist/vl-titles.js';
  * @property {string} data-vl-mobile-modal - Activeert geoptimaliseerde weergave voor op mobiele toestellen.
  * @property {string} [data-vl-mobile-modal-title=Filter] - De titel van deze search filter op mobiele toestellen indien niet gedeclareerd wordt het data-vl-title attribuut of de default genomen.
  * @property {string} data-vl-title - De titel van deze search filter.
+ * @property {string} data-vl-submit - De id van de knop die voor submit van de form van deze search filter wordt gebruikt.
```

Comment:
Heeft niets te maken met het issue dat opgelost wordt. Ik maak hiervoor een nieuwe PR aan.

Reply:
Zie PR #58.

Reply:
Blijkbaar bestaat issue [#33](https://github.com/milieuinfo/webcomponent-vl-ui-rich-data/issues/33) uit 3 issues. In deze PR worden er 2 van de 3 opgelost. Ik voorzie 3 afzonderlijke PR's om de problemen op te lossen.

Reply:
Zie PR #60 voor een oplossing voor het probleem wanneer de vl-ui-multiselect gebruikt wordt in de filter. Deze oplossing is stabieler voor huidige afnemers aangezien reeds aangeraden werd om type submit te hanteren voor de submit knop. Hierdoor is het introduceren van een extra attribuut overbodig.